src/WelcomeView.vala

```diff
@@ -19,7 +19,7 @@
 
 public class SnippetPixie.WelcomeView : Gtk.Grid {
     construct {
-        var welcome = new Granite.Widgets.Welcome ( "Snippet Pixie", _("No snippets found."));
+        var welcome = new Granite.Widgets.Welcome ( "Snippet Pixie", _("No snippets found"));
```

Comment:
The elementary standard (and my preference too) is to keep normal punctuation intact, so I'd prefer to keep the full stop.

Reply:
I find that the ending period on this specific element in the window looks very weird. It's like you're adding one to a title. I've never seen that in other apps, I'm not sure about this point :+1:

Reply:
If you look at the welcome screen for apps like Music you'll see the full stop.

Reply:
Do you have a screenshot? I can't find this

Reply:
![Screenshot from 2019-03-29 19-31-50](https://user-images.githubusercontent.com/4710/55258072-b8f2d600-5259-11e9-9473-4fa81e7126f3.png)

![Screenshot from 2019-03-29 19-33-02](https://user-images.githubusercontent.com/4710/55258084-c1e3a780-5259-11e9-9bfc-6c74ecaac067.png)

Reply:
Ok I understand. I still find that this is looking weird, but I'm going to fix this